Dockerfile

```diff
@@ -96,6 +97,8 @@ VOLUME [ "/opt/system/tmp/cache/", \
          "/home/ruby/.luarocks" \
        ]
 
+RUN dbus-uuidgen | sudo tee -a /etc/machine-id
```

Comment:
every new command in the dockerfile adds a new layer in the docker image. 
do you think we could move this to one of the other `RUN` blocks?